src/test/java/testPackage/Test1.java

```diff
@@ -7,5 +7,6 @@ public class Test1 {
 	public void test1()
 	{
 		System.out.println("print test 1");
+		System.out.println("print test 1");
```

Comment:
c 
```suggestion
		System.out.println("print test 1");
```
System.out.println("print test 2");